docs/source/getting_started/content/con_understanding-codeready-containers-configuration.adoc

```diff
@@ -0,0 +1,10 @@
+[id="understanding-codeready-containers-configuration_{context}"]
+= Understanding {prod} configuration
+
+Both the [command]`{bin}` binary and the {prod} virtual machine can be configured using the [command]`{bin} config` command.
+The [command]`{bin} config` command requires a subcommand to act on the configuration.
+The available subcommands are `get`, `set,` `unset`, and `view`.
+The `get`, `set`, and `unset` subcommands operate using named configurable properties.
```

Comment:
Is the word "operate" here really necessary? I understand the sentence without it, at least I think I do.

Reply:
"Operate on" makes the most sense.

In this context, a named configurable property is the accurate term for... well, a configurable property that has a name. These can be listed with `crc config --help`.

Reply:
> In this context, a named configurable property is the accurate term for... well, a configurable property that has a name. These can be listed with `crc config --help`.

Ok. The word "named" sounds redundant (i.e. fluff) to me. Is there a configurable property without a name? I'm assuming there isn't.